Assets/Scripts/UI/Pickup Pup/DogProfile/DogAdoptProfile.cs

```diff
@@ -0,0 +1,34 @@
+﻿/*
+ * Author: Grace Barrett-Snyder 
+ * Description: UI for displaying an individual dog's profile in the Shelter scene
+ */
+
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DogAdoptProfile : DogProfile
+{
+    [SerializeField]
+    Text priceText;
+    [SerializeField]
+    Button adoptButton;
+
+    protected override void setReferences()
+    {
+        base.setReferences();
+        rehomeButton.interactable = false;
```

Comment:
Where do these buttons get re-enabled? I feel as though a cleaner implementation of this would be to setup a panel / layer that blocks raycasts to the buttons that is dismissed / destroyed along with the profile